src/radical/pilot/raptor/master.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, cfg=None):
         self._term       = mt.Event()  # termination signal
         self._thread     = None        # run loop
 
-        self._hb_freq    = 10          # check worker heartbetas every n seconds
-        self._hb_timeout = 15          # consider worker dead after 15 seconds
+        self._hb_freq    = 1000        # check worker heartbetas every n seconds
+        self._hb_timeout = 150         # consider worker dead after 15 seconds
```

Comment:
Aymen, can you please also update comment here: `15` -> `n` (similar to the comment for previous line)